dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -67,10 +67,23 @@ endpoint.httpOptions=HTTP Options
 endpoint.httpOptions$Ref=HTTP options reference
 endpoint.httpOptions.desc=HTTP protocol options for the endpoint.
 
+endpoint.remoteIp=Remote Ip Options
```

Comment:
Make these changes globally:

- Wherever you have "IP", make the I and P both capitals.

- Wherever you have "HTTP Accessing logging" change it to a lowercase "a" so that it is "HTTP access logging".

-